pkgs/development/compilers/vyper/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, python3, fetchPypi, git, python3Packages }:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
Using `pythonXPackages` isn't allowed from `python-modules` as this will mix interpreter versions for a given package. For example, if you were to do `python37Packages.<mypkg>`, but the expression pulled from `python3[8]Packages`. Python will be mis-importing several libraries, and this will likely cause issues at runtime.

Please refer to [Python dependency handling](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#handling-dependencies) on the best practices for handling dependencies within `python-modules`. Namely, you should be importing each python package individually, and not refer to another python package set (E.g. `python3Packages`).

If you're new to nixpkgs, you're free to view some of my videos on nixpkgs and python:
- [Python packaging using nixpkgs](https://www.youtube.com/watch?v=jXd-hkP4xnU)
- [Adding a package to nixpkgs unstable](https://www.youtube.com/watch?v=fvj8H5yUKu8)

Reply:
Thanks.